package.json

```
@@ -46,6 +46,7 @@
   },
   "dependencies": {
     "compromise": "14.14.4",
+    "minimatch": "^10.0.1",
```

```suggestion
    "minimatch": "10.0.1",
```
Pin this dependency and then run CLI command `npm i` to pin this dependency in `package-lock.json` too.